packages/superset-ui-legacy-plugin-chart-markup/src/Markup.css

```diff
@@ -26,7 +26,13 @@
 .separator hr {
   border: 0;
   height: 1px;
-  background-image: linear-gradient(to right, rgba(0, 0, 0, 1), rgba(0, 0, 0, 1), rgba(0, 0, 0, 1), rgba(0, 0, 0, 0));
+  background-image: linear-gradient(
+    to right,
```

Comment:
Could simplify this (if you indeed think it's simpler) to something like:
```
background-image: linear-gradient(
    to right,
    rgba(0, 0, 0, 1) 66%,
    rgba(0, 0, 0, 0)
  );
```
example in action: https://codepen.io/rusackas-1470804400/pen/oNXvdYW

Reply:
This was done by prettier. May have to change the config.

Reply:
Not important, no worries.